velox/core/PlanNode.cpp

```diff
@@ -41,6 +41,17 @@ PlanNodePtr deserializeSingleSource(const folly::dynamic& obj, void* context) {
 PlanNodeId deserializePlanNodeId(const folly::dynamic& obj) {
   return obj["id"].asString();
 }
+
+bool containsDuplicates(const std::vector<FieldAccessTypedExprPtr>& fields) {
```

Comment:
- Looks using sort and std::adjacent_find is not proper for this case.
- Constructing an unordered_map and comparing the map size with fields size requires objects copy.

So below for loop is still used to compare duplicates.